tesseract_core/sdk/cli.py

```diff
@@ -491,6 +502,11 @@ def serve(
     else:
         ports = None
 
+    if service_names is not None:
```

Comment:
Let's check here if `--no-compose` is given and error out with a helpful message.

Reply:
done